### PR TITLE
LPS-198861 Manually delete the layout

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPS-198861

### CODE REVIEW

File: modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/security/permission/wrapper/test/CTOnDemandUserPermissionCheckerWrapperTest.java [hunk 1]
```diff
@@ -91,6 +95,8 @@ public void testHasPermission() throws Exception {
 		Assert.assertTrue(
 			_layoutPermission.contains(
 				permissionChecker, layout, ActionKeys.VIEW));
+
+		LayoutLocalServiceUtil.deleteLayout(layout);
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/security/permission/wrapper/test/CTOnDemandUserPermissionCheckerWrapperTest.java:99]: Use the injected service

**Review by pei-jung — CHANGES_REQUESTED** [review]
@brookedalton let's make the commit message more informative by adding a brief explanation on why we need to manually delete the layout.

**Comment by brookedalton** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:forward

**Comment by brookedalton** [comment]
ci:forward:force

**Pull request closed without merging**